hudi-utilities/src/test/resources/delta-streamer-config/jdbc-source-password.txt

```diff
@@ -0,0 +1 @@
+SomeTestPasswordGoesHere
```

Comment:
should we support both a password prop and a password file?  would be good to support how passwords are managed per best practices

Reply:
Sure I will add the password property